2018-10-24 09:38:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE LENTELE (ID INT NOT NULL PRIMARY KEY, NAME VARCHAR(20) [*]"; expected "FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE lentele (ID int NOT NULL PRIMARY KEY, name varchar(20) [42001-197]
2018-10-24 09:38:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "LENTELE" not found; SQL statement:
INSERT INTO lentele VALUES (1, "Petras"),(2,"Kazys"),(3,"Antanas") [42102-197]
2018-10-24 09:38:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement ")[*]"; SQL statement:
) [42000-197]
2018-10-24 09:39:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "Petras" not found; SQL statement:
INSERT INTO lentele VALUES (1, "Petras"),(2,"Kazys"),(3,"Antanas") [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:150)
	at org.h2.command.dml.Insert.prepare(Insert.java:314)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1077)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1386)
	at org.h2.server.web.WebApp.query(WebApp.java:1060)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1022)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1009)
	at org.h2.server.web.WebThread.process(WebThread.java:165)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.base/java.lang.Thread.run(Thread.java:844)
2018-10-24 09:39:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement ")[*]"; SQL statement:
) [42000-197]
2018-10-24 09:39:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO LENTELE VALUES ((1, ""Petras""),(2,""Kazys""),(3,""Antanas"") [*]"; expected "[, ::, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
INSERT INTO lentele VALUES ((1, "Petras"),(2,"Kazys"),(3,"Antanas") [42001-197]
2018-10-24 09:39:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement ")[*]"; SQL statement:
) [42000-197]
2018-10-24 09:39:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO lentele VALUES ((1, "Petras"),(2,"Kazys"),(3,"Antanas")) [21002-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:306)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1077)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1386)
	at org.h2.server.web.WebApp.query(WebApp.java:1060)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1022)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1009)
	at org.h2.server.web.WebThread.process(WebThread.java:165)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.base/java.lang.Thread.run(Thread.java:844)
2018-10-24 09:40:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO lentele VALUES ((1, 'Petras'),(2,'Kazys'),(3,'Antanas')) [21002-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:306)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1077)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1386)
	at org.h2.server.web.WebApp.query(WebApp.java:1060)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1022)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1009)
	at org.h2.server.web.WebThread.process(WebThread.java:165)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.base/java.lang.Thread.run(Thread.java:844)
2018-10-24 09:58:58 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/jonas/db/manoduombaze.db.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:307)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2506)
	at org.h2.mvstore.MVStore.panic(MVStore.java:383)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:364)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2934)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2538)
	at org.h2.engine.Database.open(Database.java:709)
	at org.h2.engine.Database.openDatabase(Database.java:286)
	at org.h2.engine.Database.<init>(Database.java:280)
	at org.h2.engine.Engine.openSession(Engine.java:66)
	at org.h2.engine.Engine.openSession(Engine.java:179)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at lt.vtvpmc.ems.jonas.testine.DbEngage.<init>(DbEngage.java:16)
	at lt.vtvpmc.ems.jonas.testine.Main.main(Main.java:6)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/jonas/db/manoduombaze.db.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 26 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/home/jonas/db/manoduombaze.db.mv.db [1.4.197/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:870)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:350)
	... 20 more
